### ISSUE TITLE
POST method requests do not support Asian characters

### ISSUE THREAD

**Issue by ryuichis** [Countly/countly-sdk-ios#89]
Upgraded to 16.06 SDK, and what worked before now is broken due to the following server error:

``` 
Parse events JSON failed [{"dur":0,"hour":14,"segmentation":{"track":"e(f0e , f4>e    d?
\/TRY (e
e$ g
h23e(gd8;i!
           f) - Kung Fu Panda 3 Official Theme Song"},"count":1,"key":"PlayTrack","timestamp":1467495918,"dow":6,"sum":0}] /i app_key=<app-key>&device_id=<device-id>&timestamp=1467495935&hour=14&dow=6&sdk_version=16.06&sdk_name=objc-native-ios&events=[{"dur":0,"hour":14,"segmentation":{"track":"周杰倫, 派偉俊\/TRY (功夫熊貓3全球主題曲) - Kung Fu Panda 3 Official Theme Song"},"count":1,"key":"PlayTrack","timestamp":1467495918,"dow":6,"sum":0}]
```

Apparently, the issue is related to the Asian characters in the request. 

Another thing I also noticed is that, this commit c0b8ce61 makes all requests with POST method, however, when I checked on the API reference http://resources.count.ly/docs/i, it is still a GET method.

**Comment by erkanyildiz** [comment]
Thanks for reporting. 

It is already fixed here: 
https://github.com/erkanyildiz/countly-sdk-ios/commit/9a1baa63b03c0db643b1fa6bb8de61d535efe5cd

It will be merged soon. I will let you know when it is merged.

**Comment by erkanyildiz** [comment]
It is merged in version 16.06.1

**Comment by ryuichis** [comment]
Nice, thanks @erkanyildiz
